### PR TITLE
Views

### PR DESCRIPTION
**Pull request by kosher9**
Changes made in this pull request :
- [x] Added some styling.

### CODE REVIEW

File: app/views/posts/show.html.erb [hunk 1]
```diff
@@ -1 +1,16 @@
...
+    <div class="post-info-head">
+      <h3>Post <%=@post.id %> by <%= @user.name %></h3>
+      <div class="post-right-info">
+        <h5>Comment : x</h5>
+        <h5>Likes : x</h5>
+      </div>
+    </div>
+
+    <p><%= @post.text %></p>
+  </div>
```
Comment (elisha2kyakpo1) [inline app/views/posts/show.html.erb:12]: - [ ] Kindly use our  `recent_comments` method in the post show view to render the most recent comments.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @kosher9, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The PR title and description looks professional 👍
- No linter errors 👍
- Descriptive commit messages 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Add your personal details in the project readme file, like your name, social links, etc.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/show.html.erb (hunk 1 above)
Comment (kosher9) [reply app/views/posts/show.html.erb:12]: Hi @elisha2kyakpo1 , sorry, I am not getting what you are talking about. We never created any `recent_comment` method. Only a method to get the three recent posts and I am using that one in my code.

Comment (elisha2kyakpo1) [reply app/views/posts/show.html.erb:12]: @kosher9, you have a function called `five_recent_comments` in the post model. That function isn't used anywhere.

Comment (kosher9) [reply app/views/posts/show.html.erb:12]: Oh you are right. Let me fix that.

Comment (kosher9) [reply app/views/posts/show.html.erb:12]: @elisha2kyakpo1 , I am done. Should I request another review?

Comment (elisha2kyakpo1) [reply app/views/posts/show.html.erb:12]: @kosher9, yes you can submit.

**Review by Davidosky007 — APPROVED** [review]
Hi @kosher9 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Proper use of Gitflow 👍 
* All views work as expected 💯 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**